components/studio/bin/hab-studio.ps1

```diff
@@ -65,6 +65,7 @@ COMMON FLAGS:
     -n  Do not mount the source path into the Studio (default: mount the path)
     -q  Prints less output for better use in scripts
     -v  Prints more verbose output
+    -w  Use a Windows studio
```

Comment:
If you don't specify this, do you get a Linux-studio-in-a-container like you do on MacOS? If so, that might be good to put in the help text.